pkgs/by-name/co/conduktor-ctl/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "conduktor-ctl";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "conduktor";
+    repo = "ctl";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-1keo9nL+vC8jYSvKkEyj2ridhKyu7ekyTJTk0/7gTHY=";
+  };
+
+  vendorHash = "sha256-HoIRw72Rxonwozqqz8z+YtZDKIJ6k5pqjf/EQFkhDik=";
+
```

```suggestion
  nativeBuildInputs = [ installShellFiles ];
```

---

```suggestion
  ldflags = [ "-X=utils.version=${version}" ];
```

https://github.com/conduktor/ctl/blob/27c023017b09145b8fa0c0ad84b0c7def4461053/utils/version.go#L3